luda-editor/server/document-macro/src/lib.rs

```diff
@@ -206,18 +207,20 @@ pub fn document(
     let impl_document = {
         let pk = {
             let pk_double_quote_content: TokenStream = ("\"".to_string()
-                + &pk_fields
-                    .iter()
-                    .map(|field| format!("#{}:{{}}", field.ident.as_ref().unwrap()))
-                    .collect::<String>()
+                + &pk_fields.iter().fold(String::new(), |mut content, field| {
+                    let _ = write!(content, "#{}:{{}}", field.ident.as_ref().unwrap());
+                    content
+                })
```

Comment:
??? white's going on here?

Reply:
Maybe clippy's new lint rule

Reply:
[format_collect](https://rust-lang.github.io/rust-clippy/master/index.html#/format_collect) in [1.73](https://github.com/rust-lang/rust-clippy/blob/master/CHANGELOG.md#new-lints)